docker/run.sh

```diff
@@ -1,14 +1,6 @@
 # For systems without journald.
 mkdir -p /var/log/journal
 
-# Copy host libsystemd into image to avoid compatibility issues. This operation
```

Comment:
Is this the reason why we are making the switch?

Reply:
Yes, these initial additions as well as current deletions are based on findings from the fluentd-gcp image here: https://github.com/GoogleCloudPlatform/k8s-stackdriver/pull/101

Reply:
Cool cool. Thanks for the clarification!